cc_library/src/sciler/README.md

```diff
@@ -9,23 +9,14 @@ This mainly consists of:
 - `device_manual.md` the manual for writing configuration files for devices
 - `LICENSE.md` the license with which this library complies
 
-## Set-up device with package.
-- create custom device script, which should inherit from `Device`, add it to `cc_library/src/scripts`
-- write configuration for the the device, according to `device_manual.md`, add it to `cc_library/src/scripts`
-- test the device script, by altering `device_test.py`, import the class on line 7, change the device class in line 15 and the config file name in line 21
+## Set-up Device
+- run `pip install -i https://test.pypi.org/simple/ scclib`
+- create custom device script, which should inherit from Device
```

Comment:
create a custom device script, with a class inheriting from the Device superclass, whose main method is called in the script